interactions/join-cave.js

```diff
@@ -3,6 +3,7 @@ const { sleep } = require('../util/sleep')
 const { transcript } = require('../util/transcript')
 const { prisma } = require('../db')
 const { getEmailFromUser } = require('../util/get-invite')
+const { scheduleHelpMeMessage } = require('../util/notify-channel')
```

Comment:
Why call the file 'notify-channel' when the exports are `scheduleHelpMeMessage` and `destroyHelpMeMessage`?